storage/innobase/row/row0log.cc

```diff
@@ -205,6 +215,138 @@ struct row_log_t {
 				log operation */
 };
 
+struct crypt_info_t {
+
+	/** Encryption algorithm */
+	Encryption::Type	encryption_type;
```

Comment:
Essentially duplicates each other with srv_encrypt_online_alter_logs. Not sure whether it needs fixing, and how

Reply:
Right now we have implemented only AES, but maybe in future there will be more algorithms to choose from?